pkg/controllers/certificate/cert_rotation_controller.go

```diff
@@ -238,7 +238,8 @@ func (c *CertRotationController) createCSRInControlPlane(clusterName string, pri
 
 	certificateSigningRequest := &certificatesv1.CertificateSigningRequest{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: csrName,
+			Name:   csrName,
+			Labels: util.ManagedByKarmadaLabels,
```

Comment:
/hold
since it's ceated in the Karmada control-plane, there is no need to add this label here.

Reply:
Good point!
@a7i do you agree with that?

Reply:
when I added this label, my intention was to exclude it from opa-gatekeepr policies. for our use-case, we still run opa at the api level (otherwise, the rollouts would pass, but propagation would fail).